pkg/cli/devicechanges.go

```diff
@@ -74,3 +95,30 @@ func runWatchDeviceChangesCommand(cmd *cobra.Command, args []string) error {
 		_ = tmplChanges.Execute(GetOutput(), in.Change)
 	}
 }
+
+func wrapPath(path string, lineLen int, tabs int) string {
+	over := len(path) % lineLen
+	linesC := len(path) / lineLen
+	var wrapped []string
+	if over > 0 {
+		wrapped = make([]string, linesC+1)
+	} else {
+		wrapped = make([]string, linesC)
+	}
+	var i int
+	for i = 0; i < linesC; i++ {
+		wrapped[i] = path[i*lineLen : i*lineLen+lineLen]
+	}
+	if over > 0 {
+		overFmt := fmt.Sprintf("%s-%ds", "%", lineLen)
```

Comment:
is this % correct ? I was expecting a value

Reply:
it's creating a formatting string, which will be used in a further `printf` - weird how it highlights it here